generators/client/templates/angular/src/test/javascript/spec/app/shared/base.component.spec.ts.ejs

```diff
@@ -0,0 +1,58 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+jest.mock('ng-jhipster');
```

Comment:
Why mocking `ng-jhipster`? `ng-jhipster` is not used here so mocking this is really redundant.

Reply:
sorry this was just a bad copy paste of the license